packages/cypher-builder/src/expressions/functions/ListFunctions.ts

```diff
@@ -77,7 +97,14 @@ class ReducerFunction extends CypherFunction {
     }
 }
 
-/** Reduce a list by executing given expression <https://neo4j.com/docs/cypher-manual/current/functions/list/#functions-reduce> */
+/** Reduce a list by executing given expression.
+ * ```cypher
+ * reduce(totalAge = 0, n IN nodes(p) | totalAge + n.age)
+ * ```
```

Comment:
Is it worthwhile as having this as an example?

Reply:
I think it is useful, considering the complexity of this statement.

Although, we should add a few more examples I think

Reply:
Sorry, I actually meant is it worth putting this under an `@example` tag so it gets rendered in the output documentation.